package.json

```diff
@@ -5,7 +5,7 @@
     "main": "index.js",
     "scripts": {
         "dev": "babel-node $(npm bin)/webpack-dev-server --config webpack.config.dev.js --port 9001 --host localhost.paypal.com --open-page demo/dev/index.htm --https --hot=false --inline=false",
-        "lint": "eslint --ext .js --ext .jsx src/ test/ *.js",
+        "lint": "eslint --ext .js --ext .jsx src/ test/ *.js --fix",
```

Comment:
I'm not so sure we should do this by default.  I've seen this wreak havoc before where linting was changed and broke the code.  Since there shouldn't be many linting errors, it seems safer to just be alerted to them and manually fixing them.

Reply:
Fair. I was just getting annoyed with the linter constantly yelling at me about my sloppy whitespace lol